index.html

```diff
@@ -272,6 +272,78 @@ <h3>Virtual Reality Gaming</h3>
 References:
 <a href="https://lists.w3.org/Archives/Public/public-webrtc/2018May/0063.html">Mailing list discussion</a>
 </section>
+<section id="no-trust-webex">
+<h3>Don't Pown My Video Conferencing </h3>
+
+<p>Cloud video conferencing systems have no need to be able to access
+  the cleartext media and text flowing through their servers.
+  Some of these conferencing services desire to be able to promote trust
+  by explicitly showing they do not have access to contents of their
+  users' calls. They are trusted to connect the right people to the
+  conference and to route the packets but they are not trusted to access
+  the audio and video media or text in the call.</p>
+
+<p>Solutions to this problem fall into two major categories: one where
+  the JavaScript comes from a source trusted to see the media contents,
+  and one where it does not:</p>
+
+<h4>Untrusted JavaScript Cloud Conferencing</h4>
+
+<p>There are many cases where a system such as WebEx is trusted to
+  connect the members of a conference but has no need to access the
+  contents of the conference. This is true of the majority of
+  conferencing systems on the web today.  Just to highlight the
+  scope of this requirement, there are more minutes of WebRTC
+  that are used in conferences where the servers have no need
+  to access the contents (e.g. where audio is forwarded rather than
+  mixed) than any other use of WebRTC audio by orders of magnitude.
+  This is one of the primary use case for WebRTC audio and accounts
+  for billions of minutes per month of potential use of WebRTC.
+</p>
+
+<p>In this use case, the JavaScript comes from the operator of the
+  conference bridge. The isolated media features of WebRTC prevent
+  the JavaScript from accessing the media and the identity features
+  are used to provide a user interface that allows the user to know
+  it connected to the correct conference. The goal is for the end
+  users to be able to see the contents, but the web service that
+  provides the JS and the media switching bridges and Selective
+  Forwarding Units (SFUs) cannot access the contents (audio, video,
+  text). Some metadata about the audio, such as the power
+  levels of the audio media, are encrypted end-to-end and are 
+  therefore not accessible to the media server.
+</p>
+
+<p>A possible solution this problem is the browser to negotiate end to
+  end encryption keys which are not revealed to the JavaScript.
```

Comment:
Could mention MLS in the browser here as potential solution. I have wrote this scenario down in the federation draft

Reply:
The use case doesn't get into much detail on security requirements.  For example, properties such as non-repudiation, perfect forward secrecy (PFS) or post-compromise security (PCS) are not mentioned. The MLS architecture draft does get into this. 

For example, draft-ietf-mls-architecture Section 3.2.25 discusses Non-Repudiation vs Deniability:

   As described in Section 4.4, MLS provides strong authentication
   within a group, such that a group member cannot send a message that
   appears to be from another group member.  Additionally, some services
   require that a recipient be able to prove to the messaging service
   that a message was sent by a given client, in order to report abuse.
   MLS supports both of these use cases.  In some deployments, these
   services are provided by mechanisms which allow the receiver to prove
   a message's origin to a third party (this if often called "non-
   repudiation"), but it should also be possible to operate MLS in a
   "deniable" mode where such proof is not possible.

Reply:
Filed Issue https://github.com/w3c/webrtc-nv-use-cases/issues/37 relating to security requirements.